src/parser/serializer.ts

```diff
@@ -410,7 +419,8 @@ function serializeCode(
   checkKeys = false,
```

Comment:
perhaps someday we should consider a context object for bson serialization too.  9 position arguments, some with defaults, feels excessive.  just a thought